x-pack/plugins/reporting/common/index.ts

```diff
@@ -6,6 +6,7 @@
 
 import { LayoutSelectorDictionary } from './types';
 
+export * as constants from './constants';
```

Comment:
Did you mean to also alter the reporting plugin stuff as well?

Reply:
Yes, Reporting changes had to go in as part of this PR.

Most of it involves changing where things are defined in Reporting to allow the example plugin to succeed the webpack build in CI. The example plugin is all UI code, and it can only import things from Reporting if they also originate in UI code.

At first, I thought the example plugin UI would be able to import code from Reporting "common" but that didn't work out: it had to be defined in `reporting/public`

Reply:
Noted this in the PR description